sdk/storage/azure-storage-file-share/swagger/README.md

```diff
@@ -644,21 +648,6 @@ directive:
     $["x-ms-parameter-location"] = "method";
 ```
 
-### Add the CustomFileAndDirectoryListingDeserializer attribute
```

Comment:
Why did we delete this? It looks like the custom deserializer is still there

Reply:
Deleted this and added it to the ShareStorageCustomization clas. This way no longer works with the new generator